script/Ciel/Day 11/11_01_CIEL11_1C.txt

```diff
@@ -174,7 +177,7 @@ The sluggish feeling in my limbs that I woke up with hasn't improved at all.
 [sha:57fa69d0c922f2612865bf63be83c1a689f5578c]{
 -- Page 61, Offset 29174.
 -- 根本的な解決が必要だ。
-I need a radical solution.
+I require a more decisive solution.
```

Comment:
Looking at it now, it would almost be better with "final solution" if it wasn't for the fact that there are some unfortunate implications. Well, sometimes that's how life is. Nothing to do about it.